interfaces/Matlab_Toolbox_Revamp/Class/Kinetics.m

```diff
@@ -0,0 +1,590 @@
+classdef Kinetics < handle
+    
+    properties
+        owner
+        kin_id
+        Kc % equilibrium constant
+        Kf % forward reaction rate
+        Kr % reverse reaction rate
+        dH % enthalpy of reaction
+        dS % entropy of reaction
+        dG % gibbs free energy of reaction
+    end
+
+    methods
+        %% Kinetics class constructor
+
+        function kin = Kinetics(ph, src, id, n1, n2, n3, n4)
+            checklib;
+            % indices for bulk phases in a heterogeneous mechanism
+            inb1 = -1;
+            inb2 = -1;
+            inb3 = -1;
+            inb4 = -1;
+            if nargin == 2
+                id = '-';
+            end
+            kin.owner = 1;
```

Comment:
What's the purpose of this attribute?

Reply:
Owner property for Kinetics, ThermoPhase, and Transport were relics from the old interface. I don't see them used anywhere, so I suppose it's fine to delete it.

Reply:
If there are leftovers from the old class structure, please be proactive about removing them. In fact, there's no particular reason to stick with the old way at all, and as Ingmar has said, we'd like to have things more closely match the structure of the Python interface.